dcpy/lifecycle/ingest/run.py

```diff
@@ -49,6 +50,8 @@ def run(
     if not skip_archival:
         recipes.archive_dataset(config, staging_dir / config.filename, latest=latest)
 
+    return config
```

Comment:
This change isn't used in recipes test after latest refactor but still is a reasonable thing - after archival, return the config